internal/commitinfo/commitinfo.go

```diff
@@ -68,11 +68,11 @@ func ParseCommitInfo(commitMessage string) (CommitInfo, error) {
 	}
 
 	matches := parseCommitInfoFromCommitMessageRegex.FindStringSubmatch(convInfo.Message)
-	if matches == nil {
-		return CommitInfo{}, errors.Wrap(ErrNoMatch, fmt.Sprintf("commit message '%s' does not match expected message structure", convInfo.Message))
+	if matches == nil && !convInfo.HasField(FieldReleaseIntent) {
+		return CommitInfo{}, errors.Wrap(ErrNoMatch, fmt.Sprintf("commit message '%s' does have a Release-intent field and did not match expected message structure", convInfo.Message))
```

Comment:
In this case id does **not** have a release intent, right?

Reply:
Ah 🤦  Fixing!